FinancialTracker/ViewControllers/HomeViewController.swift

```diff
@@ -86,14 +96,32 @@ class HomeViewController: UIViewController {
     }
     
     @objc func sendMail() {
+        let alertController = UIAlertController(title: "Support", message: nil, preferredStyle: .actionSheet)
+        alertController.addAction(UIAlertAction(title: Support.addExpense.rawValue, style: .default, handler: setComposerMessage))
+        alertController.addAction(UIAlertAction(title: Support.refundMoney.rawValue, style: .default, handler: setComposerMessage))
+        alertController.addAction(UIAlertAction(title: "Cancel", style: .cancel))
+        alertController.popoverPresentationController?.barButtonItem = self.navigationItem.leftBarButtonItem
+        
+        present(alertController, animated: true)
+    }
+    
+    private func setComposerMessage(action: UIAlertAction) {
         guard MFMailComposeViewController.canSendMail() else {
             assertionFailure("Mail services are not available")
             return
         }
         
         let composer = MFMailComposeViewController()
         composer.mailComposeDelegate = self
-        composer.setToRecipients(["support_financialTracker@gmail.com"])
+        composer.setToRecipients([Support.email])
+        switch action.title {
+        case Support.addExpense.rawValue:
+            composer.setSubject("Failed adding expenses")
```

Comment:
Can you have the strings here come from the `Support` enum directly - add a property with a suitable name and switch on `self` and so on.